### PR TITLE
Fix JS error mapping null

### PR DESCRIPTION
**Pull request by kelvin-chappell**
I was too hasty with the last fix: #112 
It just pushed the error on a bit.

### CODE REVIEW

**Comment by janua** [comment]
:+1:

File: public/components/CampaignAnalytics/Analytics/CampaignUniquesChart.js [hunk 1]
```diff
@@ -9,16 +9,17 @@ export default class CampaignUniquesChart extends React.Component {
 
   render () {
 
-    console.log(this.props.data);
-
-    let data = this.props.data.map( (item) => {
+    let data = [];
+    if (this.props.data) {
```
Comment (LATaylor-guardian) [inline public/components/CampaignAnalytics/Analytics/CampaignUniquesChart.js:13]: I'm not sure we want to show the graph if there is no data. Can we add:
```
if (!this.props.data) return null;
```
as the first line of the render method instead?

Comment (kelvin-chappell) [reply public/components/CampaignAnalytics/Analytics/CampaignUniquesChart.js:13]: @LATaylor-guardian Just saw your comment a second too late!  I'll do it in another PR

Comment (LATaylor-guardian) [reply public/components/CampaignAnalytics/Analytics/CampaignUniquesChart.js:13]: @kelvin-chappell of course, no worries :)

**Pull request merged**